src/common/snippets/include/snippets/runtime_configurator.hpp

```diff
@@ -126,7 +126,7 @@ class RuntimeConfigurator {
         std::vector<int64_t> ptr_increments;
         std::vector<int64_t> finalization_offsets;
     };
-    static UnifiedLoopInfoRtParams compute_runtime_params(const lowered::UnifiedLoopInfoPtr& unified_loop_info);
+    static UnifiedLoopInfoRtParams get_runtime_params(const lowered::UnifiedLoopInfoPtr& unified_loop_info);
```

Comment:
Minor: what's about to rename to `get_loop_runtime_params`?

Reply:
Good idea, done, thanks